SPECS/openssl/openssl.spec

```diff
@@ -476,6 +476,10 @@ install -m644 %{SOURCE9} \
 %ldconfig_scriptlets libs
 
 %changelog
+* Fri Dec 01 2023 Tobias Brick <tobiasb@microsoft.com> - 3.1.4-1
```

Comment:
If you aren't bumping the release version, can you merge the note with the previous changelog entry. Will look very odd / harder to triage to have multiple `3.1.4-1` entries when this package is eventually officially published.

Reply:
We decided to remove this changelog update altogether, since they're minor but I like the idea in general and will do that going forward.